scripts/getArchive.py

```diff
@@ -158,6 +158,11 @@ def convert_timestamps(ent):
     default="bus",
     help='Feed to retrieve. Defaults to "bus"',
 )
+parser.add_argument(
+     "--object-prefix",
+    dest="object-prefix",
```

Comment:
nitpick(non-blocking): I'd prefer this to be a valid python variable name, it seems that the rest of the code accesses values through the index operator so it doesn't _really_ matter, but it's possible to access the values via the dot operator if the `dest` is a valid variable name (the `-` is invalid IIRC)

Reply:
Fixed in f8262f7.